legesher.yml

```
@@ -0,0 +1,12 @@
+---
+name: Up-For-Grabs.net
+desc: The bridge between the words we speak and the code we write.
+site: https://github.com/legesher/legesher
+tags:
+- html/css
```

This value should be split into two separate values:

```suggestion
- html
- css
```